src/util.js

```diff
@@ -109,6 +109,15 @@ export async function getConnection ({
   return xo
 }
 
+export async function testConnection ({
+  credentials
+}) {
+  await getConnection({credentials}).then(
+    async (res) => {
+      await res.close()
+    })
+}
```

Comment:
You do not need to use async functions, promises are enough:

```js
export const testConnection = opts => getConnection(opts).then(connection => connection.close())
```

But you can use async functions if you feel more comfortable:

```js
export async function testConnection (opts) {
  const connection = await getConnection(opts)
  await connection.close()
}
```